corehq/apps/hqadmin/management/commands/stale_data_in_es.py

```diff
@@ -228,8 +227,8 @@ def _get_sql_form_data_for_db(db, run_config):
         matching_forms = XFormInstanceSQL.objects.using(db).filter(
             received_on__gte=run_config.start_date,
             received_on__lte=run_config.end_date,
-            # Exclude deleted
-            state=F('state') + F('state').bitand(XFormInstanceSQL.DELETED)
+            # Only check for "normal" forms
+            state=XFormInstanceSQL.NORMAL
```

Comment:
this seems reasonable to me though archived forms might also be in ES? I can't remember. :man_shrugging:

Reply:
I'll run a test and add archived forms if it seems like they're supposed to be in ES. It's kind of tricky because this may have changed over time so there are actually forms of all types in ES from long ago, even if we aren't actively putting them there anymore